### PR TITLE
Updated readme

### LINKED ISSUE #1362: Flesh out the repo readme
Can add things like our CI status, coverage reports, links to docker images, links to twitter etc.

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds some badges, logo, and emojies to the Repo readme. No actual content was changed other than this header. The screenshot below shows this update:

image

close #1362

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,52 @@
...
-## Documentation
+![GitHub](https://img.shields.io/github/license/UMAprotocol/protocol)
+![GitHub last commit](https://img.shields.io/github/last-commit/UMAprotocol/protocol)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/UMAprotocol/protocol)
+![GitHub contributors](https://img.shields.io/github/contributors-anon/UMAprotocol/protocol)
+
+[![Generic badge](https://img.shields.io/badge/homepage-view-red.svg)](https://umaproject.org/)
+[![Generic badge](https://img.shields.io/badge/slack-join-green.svg)](https://join.slack.com/t/umaprotocol/shared_invite/zt-7mtxxds5-OIhE~q_WkwGCVNrq0~G~rg)
+[![Generic badge](https://img.shields.io/badge/send-email-blue.svg)](mailto:hello@umaproject.org)
+![Twitter Follow](https://img.shields.io/twitter/follow/UMAprotocol?label=follow%20%40UMAprotocol&style=social)
```
Comment (mrice32) [inline README.md:16]: This seems to link to a page with just this image on it -- maybe we should link this image to twitter?

File: README.md [hunk 2]
```diff
@@ -1,44 +1,52 @@
...
-  Please use Slack for all technical questions and discussions.
-- [Email](mailto:hello@umaproject.org): for anything non-technical.
+[![<UMAprotocol>](https://circleci.com/gh/UMAprotocol/protocol.svg?style=shield)](https://app.circleci.com/pipelines/github/UMAprotocol/protocol)
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/umaprotocol/voting)
 
-## Documentation
+![GitHub](https://img.shields.io/github/license/UMAprotocol/protocol)
+![GitHub last commit](https://img.shields.io/github/last-commit/UMAprotocol/protocol)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/UMAprotocol/protocol)
+![GitHub contributors](https://img.shields.io/github/contributors-anon/UMAprotocol/protocol)
```
Comment (mrice32) [inline README.md:11]: Maybe link to our contributor page?

File: README.md [hunk 3]
```diff
@@ -1,44 +1,52 @@
...
-  [invite link](https://join.slack.com/t/umaprotocol/shared_invite/enQtNTk4MjQ4ODY0MDA1LTM4ODg0NGZhYWZkNjkzMDE4MjU0ZGFlYWQzZTFiZWFlZjI2NDE4OGI2NWY3OTdhYjYyZjg0MjAzMTgwODVhZTE).
-  Please use Slack for all technical questions and discussions.
-- [Email](mailto:hello@umaproject.org): for anything non-technical.
+[![<UMAprotocol>](https://circleci.com/gh/UMAprotocol/protocol.svg?style=shield)](https://app.circleci.com/pipelines/github/UMAprotocol/protocol)
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/umaprotocol/voting)
 
-## Documentation
+![GitHub](https://img.shields.io/github/license/UMAprotocol/protocol)
+![GitHub last commit](https://img.shields.io/github/last-commit/UMAprotocol/protocol)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/UMAprotocol/protocol)
```
Comment (mrice32) [inline README.md:10]: Maybe link to the list of commits?

File: README.md [hunk 4]
```diff
@@ -1,44 +1,52 @@
...
-- [Slack](https://umaprotocol.slack.com): to join, use this
-  [invite link](https://join.slack.com/t/umaprotocol/shared_invite/enQtNTk4MjQ4ODY0MDA1LTM4ODg0NGZhYWZkNjkzMDE4MjU0ZGFlYWQzZTFiZWFlZjI2NDE4OGI2NWY3OTdhYjYyZjg0MjAzMTgwODVhZTE).
-  Please use Slack for all technical questions and discussions.
-- [Email](mailto:hello@umaproject.org): for anything non-technical.
+[![<UMAprotocol>](https://circleci.com/gh/UMAprotocol/protocol.svg?style=shield)](https://app.circleci.com/pipelines/github/UMAprotocol/protocol)
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/umaprotocol/voting)
 
-## Documentation
+![GitHub](https://img.shields.io/github/license/UMAprotocol/protocol)
+![GitHub last commit](https://img.shields.io/github/last-commit/UMAprotocol/protocol)
```
Comment (mrice32) [inline README.md:9]: Maybe link to the last commit (or list of commits)?

File: README.md [hunk 5]
```diff
@@ -1,44 +1,52 @@
...
 
-## Contact
+<div style="text-align:center"><img src="./documentation/Logo.png" /></div>
 
-- [Slack](https://umaprotocol.slack.com): to join, use this
-  [invite link](https://join.slack.com/t/umaprotocol/shared_invite/enQtNTk4MjQ4ODY0MDA1LTM4ODg0NGZhYWZkNjkzMDE4MjU0ZGFlYWQzZTFiZWFlZjI2NDE4OGI2NWY3OTdhYjYyZjg0MjAzMTgwODVhZTE).
-  Please use Slack for all technical questions and discussions.
-- [Email](mailto:hello@umaproject.org): for anything non-technical.
+[![<UMAprotocol>](https://circleci.com/gh/UMAprotocol/protocol.svg?style=shield)](https://app.circleci.com/pipelines/github/UMAprotocol/protocol)
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/umaprotocol/voting)
```
Comment (mrice32) [inline README.md:6]: Maybe link here? https://hub.docker.com/repository/docker/umaprotocol/voting

File: README.md [hunk 6]
```diff
@@ -1,44 +1,52 @@
...
 
-## Documentation
+![GitHub](https://img.shields.io/github/license/UMAprotocol/protocol)
```
Comment (mrice32) [inline README.md:8]: Maybe link to our license file...?

**Comment by chrismaree** [comment]
> This is awesome! I can't believe this is free. My primary comment is that these badges _look like_ links (at least to me), but when clicked, some just take you to a page showing the image. I think it might be a slightly nicer experience if they all linked to relevant content (or just not link at all)? Thoughts?

Yup. I actually thought the way I had done it was hyperlinked but I guess I messed it up. Have updated all of them to appropriate links  and have tested all of them :)

File: README.md (hunk 1 above)
Comment (chrismaree) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (chrismaree) [reply README.md]: done

Comment (chrismaree) [reply README.md]: We could also add contributor guidelines to our repo if we wanted at some point? Right now there are no instructions for community members on how they should contribute code if they wanted to.

Comment (mrice32) [reply README.md]: Agreed, probably not a big deal until we get more community contribution interest, though.

**Pull request merged**